src/components/hooks/useWS.ts

```diff
@@ -15,13 +16,29 @@ const useWS = <T extends TSocketEndpointNames>(name: T) => {
     const send = useCallback(
         async (data?: Parameters<typeof apiManager.augmentedSend<T>>[1]) => {
             setIsLoading(true)
-            try {
-                const response = await apiManager.augmentedSend(name, data)
-                setData(response[name] as TSocketResponseData<T>)
-            } catch (e) {
-                setError(e)
-            } finally {
-                setIsLoading(false)
+            const readyState = parseInt(sessionStorage.getItem('websocket_ready_state'))
```

Comment:
Constant is not used

Reply:
removed session storage so resolving this